spec/serializers/export/own_post_serializer_spec.rb

```diff
@@ -1,12 +1,54 @@
 describe Export::OwnPostSerializer do
-  let(:post) { create(:status_message_with_photo) }
-  subject(:json_output) { Export::OwnPostSerializer.new(post).to_json }
-
-  # TODO: rewrite this spec
-  it { is_expected.to include %("guid":"#{post.guid}") }
-  it { is_expected.to include %("text":"#{post.text}") }
-  it { is_expected.to include %("public":#{post.public}) }
-  it { is_expected.to include %("author":"#{post.diaspora_handle}") }
-  it { is_expected.to include %("entity_type":"#{post.type.underscore}") }
-  it { is_expected.to include %("created_at":"#{post.created_at.iso8601}) }
+  let(:author) { FactoryGirl.create(:user_with_aspect).person }
+
+  before do
+    author.owner.share_with(FactoryGirl.create(:person), author.owner.aspects.first)
+  end
+
+  it_behaves_like "it is federation entity serializer" do
+    let(:object) { create(:status_message_with_photo, author: author) }
+  end
+
+  let(:json) { Export::OwnPostSerializer.new(post, root: false).to_json }
+
+  context "with private post" do
+    let(:post) { create(:status_message_in_aspect, author: author) }
+
+    before do
+      expect(post.subscribers).not_to be_empty
```

Comment:
Well, I put  this string here for "tests for test" purpose again. If we don't do that, then we should remove it. Otherwise, it makes sense for both test since `doesn't include remote pods subscriptions` test theoretically could be false positive in case when there is no subscribers.